src/Sdk/Coverage/Coverage.ts

```diff
@@ -21,7 +22,7 @@ export default class CoverageSdk {
     async list(options: CoverageSearchOptions = {}): Promise<CoverageListResponse> {
         const { jsonQuery, page, pageSize, sortOrder } = options;
         const response = await this.apiClient.get<CoverageListResponse>(
-            this.apiClient.buildUrl(this.sdkUrl),
+            buildUrl(this.apiClient.baseUrl, this.sdkUrl),
```

Comment:
> `this.sdkUrl`

There is an advantage of using `appRouting` object directly without destructureing it -- you can trace the usage of properties down to every API call. But we lose that once a value is destructured and reassigned. 

Is it worth to introduce all the intermediate properties and instance state when we can just use `routing.coverageUrl`?

Reply:
Yes, looks better this way and it's easier to track usages as you've pointed out. Done with 69de8e1706387578b8d6dd63557c8fd60e4fb194